### DISCUSSION TITLE
Walk through for setting  up Windows Chia?

### DISCUSSION THREAD

**Discussion by forkmiester** [Chia-Network/chia-blockchain#728] [Q&A]
Hi everyone,

I'm a complete newb to the Chia blockchain program. I do understand the concepts of how it works, but not in properly setting everything up. 

Does someone have a walkthrough on how to setup Chia, how to setup your wallet, how to create plots, trading, and how keys work? I've been attempting to farm plots for several days and keep on getting errors...and I have zero clue why. :(

Any help would be much appreciated.

**Comment by BrandtH22** [comment]
Since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

Note the beginners guides can be found on the docs site here:
https://docs.chia.net/docs-home/

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
